src/constants.tsx

```diff
@@ -117,3 +117,5 @@ export const MOSQUITO_COLLECTION_CODE = 'Mosquito Collection';
 export const PRACTITIONER_CODE = {
   text: 'Community Health Worker',
 };
+/** Field to sort plans by */
+export const SORT_FIELD = 'plan_effective_period_start';
```

Comment:
Do you think it makes sense to name this something more specific like to mean "sort by effective period start"?